pandas/core/strings.py

```diff
@@ -3040,20 +3040,27 @@ def encode(self, encoding, errors="strict"):
     """
 
     @Appender(
-        _shared_docs["str_strip"] % dict(side="left and right sides", method="strip")
+        _shared_docs["str_strip"]
+        % dict(side="left and right sides", method="strip", position=None)
```

Comment:
Doesn't `strip` remove both leading and trailing characters? Should this be
```suggestion
        % dict(side="left and right sides", method="strip", position="leading and trailing")
```
?

Reply:
Oh hell. Yes, it should have both leading and trailing. Fixing now and will push. Thank you.